sdk/storage/azure-storage-queue/azure/storage/queue/_encryption.py

```diff
@@ -520,7 +525,11 @@ def _generate_AES_CBC_cipher(cek, iv):
     return Cipher(algorithm, mode, backend)
 
 
-def _validate_and_unwrap_cek(encryption_data, key_encryption_key=None, key_resolver=None):
+def _validate_and_unwrap_cek(
+        encryption_data: _EncryptionData,
+        key_encryption_key: object = None,
+        key_resolver: callable = None
```

Comment:
Want to call out that in the docstring I've seen `func` and `function`, but after some searching on the interwebs it seems like the general consensus is `callable` for functions? I can make a note here to go back and update the docstrings (at the very least, unify them to either `func`, `function`, or `callable`)

Reply:
Want to use `Callable` imported from `typing`
Callable[[int], str]
Here int is the inputs, str is the outputs

Reply:
`Callable[str, bytes]`

Reply:
Use `typing.Callable[str, bytes]`

Reply:
Return is `bytes`